.gitignore

```
@@ -113,3 +113,6 @@ ENV/
 
 #kubernetes stuff
 kubernetes/jobs/
+
+# Version files
+*_version.py
```

Better be specific, right?
```suggestion
src/eddymotion/_version.py
```